product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.10-Setting-a-request-payload-with-a-script/src/test/java/org.wso2.carbon.ei.scenario.test/AlterRequestWithScriptTest.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *      http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.ei.scenario.test;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.esb.scenario.test.common.http.HTTPUtils;
+import org.wso2.carbon.esb.scenario.test.common.ScenarioTestBase;
+
+import java.io.IOException;
+import javax.xml.stream.XMLStreamException;
+
+
+ //This class is to test if xml payload can be enriched before it goes to the backend server by using script mediator.
+public class AlterRequestWithScriptTest extends ScenarioTestBase {
+
+     private  String request =
```

Comment:
This could be a private static final string. A static final string should be in upper case. Shall we correct it?

Reply:
Done.